ui/packages/catalog-ui-search/src/main/webapp/component/time-settings/time-settings.view.js

```diff
@@ -55,18 +57,110 @@ module.exports = Marionette.LayoutView.extend({
             })
         }));
     },
+    setupTimeZone() {
+        var timeZone = user.get('user').get('preferences').get('timeZone');
+
+        this.propertyTimeZone.show(new PropertyView({
+            model: new Property({
+                label: 'Time Zone',
+                value: [timeZone],
+                enum: [{
+                    label: 'UTC, +00:00',
```

Comment:
Might be worth pushing this to the middle or having a duplicate there, I was a bit surprised during testing when looking for `+00:00` between `-01:00` and `+01:00`.